Gems/RecastNavigation/Code/Include/RecastNavigation/RecastNavigationPhysxProviderBus.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/ComponentBus.h>
+#include <Misc/RecastHelpers.h>
+
+namespace RecastNavigation
+{
+    //! The interface for @RecastNavigationPhysXProviderRequestBus
+    class RecastNavigationPhysXProviderRequests
```

Comment:
Is it a good name to say `PhysXProvider`? Since the user can implement this EBUS and build his own logic of providing geometry data to Recast without using PhysX, I think a more generic name like `RecastNavigationGeometryProviderRequests` is better

Reply:
Oh, that is a good point. I will rename it into a generic ebus name that can extend beyond PhysX.

Reply:
Renamed for clarity into RecastNavigationProviderRequestBus